src/main/java/org/testng/ITestNGMethod.java

```diff
@@ -33,30 +32,12 @@
   void setTestClass(ITestClass cls);
 
   /**
-   * @return the corresponding Java test method.
-   * @deprecated This method is deprecated and can now return null. Use
-   * getConstructorOrMethod() instead.
-   */
-  @Deprecated
-  Method getMethod();

```

Comment:
@krmahadevan , please keep this api, as it's used by testng-remote:
https://github.com/testng-team/testng-remote/blob/master/remote/src/main/java/org/testng/remote/strprotocol/TestResultMessage.java#L134-L139